testsuite/tests/parsetree/source_jane_street.ml

```diff
@@ -1616,3 +1616,11 @@ val triangle_10 : int = 55
 (* quotations syntax *)
 
 (* Test will only be added once quotations work end-to-end. *)
+
```

Comment:
Could you add several tests of syntax that users might want to write but rejected? See `typing-modes/syntax-error.ml` for examples.